lib/ios/RNNControllerFactory.m

```diff
@@ -101,8 +101,9 @@ -(RNNTabBarController*)createTabs:(RNNLayoutNode*)node {
 	NSMutableArray* controllers = [NSMutableArray new];
 	for (NSDictionary *child in node.children) {
 		UIViewController* childVc = [self fromTree:child];
-		
-		UITabBarItem* item = [[UITabBarItem alloc] initWithTitle:@"A Tab" image:nil tag:1];
+
+		NSString* title = child[@"children"][0][@"data"][@"navigationOptions"][@"title"] ?: @"A Tab";
```

Comment:
The access to `[0]` freaks me out a little bit here. What do you think about pulling this line out into something that checks for the first object here, rather than accessing it blindly?

Reply:
Sure thing 👍